script.js

```diff
@@ -74,7 +85,16 @@ function displayBooks() {
   });
 }
 
+function checkBorder() {
+  if (booksObj.books.length !== 0) {
```

Comment:
you could say if === 0 here to make it simpler if you want

Reply:
Okay. I think I will leave it as it is